rosco-web/config/rosco.yml

```diff
@@ -5,7 +5,9 @@ rosco:
   configDir: /some/path/to/config/packer
   jobs:
     local:
-      timeoutMinutes: 20
+      timeoutMinutes: 30
```

Comment:
Ok, so is this resolved?
@rguthriemsft This LG2U? And the companion spinnaker/spinnaker pr as well?